### ISSUE TITLE
Neovide crashes

### ISSUE THREAD

**Issue by AguirreIF** [neovide/neovide#2051]
**Describe the bug**
After opening Neovide and change the focus to another window Neovide crahes.

**To Reproduce**
1. Open a terminal widow (or any other application) and then open Neovide.
2. With the mouse change the focus from Neovide to the terminal window.
3. Neovide crashes.
4. The file `neovide_backtraces.log` is created.

**Expected behavior**
Neovide should not crash when it looses the focus.

**Desktop (please complete the following information):**
 - OS: GNU/Linux Debian trixie
 - Neovide Version: 0.11.2
 - Neovim Version: 0.7.2

**Content of neovide_backtraces.log:**

```
2023-09-25 09:15:36 - Neovide panicked with the message 'Focus Gained Failed: NeovimError(Some(0), "Invalid method: nvim_ui_set_focus")'. (File: src/bridge/ui_commands.rs; Line: 150, Column: 47)
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>

2023-09-25 09:15:36 - Neovide panicked with the message 'Focus Lost Failed: NeovimError(Some(0), "Invalid method: nvim_ui_set_focus")'. (File: src/bridge/ui_commands.rs; Line: 147, Column: 48)
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>

2023-09-25 09:15:37 - Neovide panicked with the message 'Mouse Input Failed: NeovimError(Some(1), "invalid button or action")'. (File: src/bridge/ui_commands.rs; Line: 71, Column: 18)
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>

2023-09-25 09:15:37 - Neovide panicked with the message 'Focus Lost Failed: NeovimError(Some(0), "Invalid method: nvim_ui_set_focus")'. (File: src/bridge/ui_commands.rs; Line: 147, Column: 48)
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>

2023-09-25 09:15:37 - Neovide panicked with the message 'Could not send serial ui command: SendError { .. }'. (File: src/bridge/ui_commands.rs; Line: 265, Column: 22)
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>

2023-09-25 09:15:37 - Neovide panicked with the message 'called `Result::unwrap()` on an `Err` value: SendError { .. }'. (File: src/event_aggregator.rs; Line: 51, Column: 28)
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>
  16: <unknown>
  17: <unknown>
  18: <unknown>
  19: <unknown>
  20: <unknown>
  21: __libc_start_main
  22: <unknown>
```

**Comment by fredizzimo** [comment]
You need to update your Neovim version. Our code specifices the wrong minimum requirements, which is 0.8. 

But I suggest that you upgrade to 0.9.2, since we will soon be requiring that.

**Comment by AguirreIF** [comment]
> You need to update your Neovim version. Our code specifices the wrong minimum requirements, which is 0.8.
> 
> But I suggest that you upgrade to 0.9.2, since we will soon be requiring that.
> 
> **Edit:** The master already requires 0.9.2, I did not realize that nvim_ui_set_focus was that new

You are right, sorry for that. I'm going to build nvim from source, in Debian the last available version is 0.7.2.
